Themes/README.md

```diff
@@ -2,19 +2,27 @@
 
 [Semantic colorization was added to the C/C++ Extension in version 0.24.0](https://devblogs.microsoft.com/cppblog/visual-studio-code-c-c-extension-july-2019-update/). At the time, colorization in VS Code was purely syntactic/lexical and leveraged TextMate grammar to associate named 'scopes' with syntactic elements. Themes and settings can be used to associate colors with these scopes. Our original implementation of semantic colorization leveraged the same system of associating colors with named scopes. But, some tokens that can be colored by semantic colorization in C/C++ did not have existing analogs in VS Code's TextMate grammar. So, new named scopes are required. Because these scopes were new, existing themes did not include colors for them either.
 
-We created C/C++ Extension UI Themes to closely match Visual Studio themes and include colors for many of the new scopes.
+We created C/C++ Extension UI Themes to closely match Visual Studio themes and include colors for many of the new scopes. The old themes from before Visual Studio 2019 can be found alongside new themes that more closely resemble the Enhanced themes.
 
 VS Code has since provided an API for semantic colorization. The C/C++ Extension has transitioned from its own implementation to this new API. These themes now include colors for some of the new semantic token scopes.
 
 ## Example
 
 Light Theme
 
-![Light Theme example](https://raw.githubusercontent.com/microsoft/vscode-cpptools/main/Themes/assets/light.png)
+![Light Theme 2019 example](./assets/light.png)
 
 Dark Theme
 
-![Dark Theme example](https://raw.githubusercontent.com/microsoft/vscode-cpptools/main/Themes/assets/dark.png)
+![Dark Theme 2019 example](./assets/dark.png)
+
+Classic Light Theme
```

Comment:
What does "Classic" denote here?  Usually, I'd expect "Classic" to refer to something that's an older option.  In this case, it's the newer (2022-aligned) themes that are being referred to as Classic, right?   How about referring to the new themes as "VS 2022", and the older themes as "VS 2019".  That would seem to line up fairly accurately.

Reply:
Ah, I got that reversed.  The older one is the "Classic" once.  Instead of "Classic", though, let's go with 2019 and 2022.

Reply:
I am still certain that it was Visual Studio 2019 that added the new themes. Are you sure they were added in 2022 and not 2019?

Reply:
The "Enhanced" theme was added in VS 2019: see https://devblogs.microsoft.com/cppblog/productivity-improvements-for-c-new-default-colorization-template-argument-filtering-in-call-stack-window-and-intellicode-on-by-default/ .

Isn't there just a 2017 and 2019/Enhanced theme? Or is there a 3rd one?

Reply:
As Sean pointed out, the new themes were not added until 2019 16.3 Preview 2.  (So, significantly after the release of 2019).  We called the "Classic" themes "2019" as they were the themes in use in 2019 at the time, though had not been changed since VS 2017..

The C++ settings in VS refer to these themes as "2017" and "Enhanced":

![image](https://user-images.githubusercontent.com/49173979/193699816-971adbec-9b59-4360-aa52-0055fa8f3e10.png)

However, for C#, they refer to the Enhanced theme as 2019.

![image](https://user-images.githubusercontent.com/49173979/193699914-c096d911-daf9-406b-936d-ff168de4a9bb.png)

That changed at some point, as they used to refer to them as "enhanced":

![image](https://user-images.githubusercontent.com/49173979/193700509-cfd31339-915e-490f-b1e8-11eccd50adf6.png)

IMHO, we should either go one of these options:

* 2017 and 2022
* 2017 and 2019
* 2017 and Enhanced

Those are listed in my order of preference.  :)

Reply:
The actual themes in `package.json` that are exposed to our users are called "Visual Studio Light - C++" and "Visual Studio Dark - C++".  Those cannot change.  In the documentation we should refer to the themes by the names that will be shown in the theme picker.  We have essentially 2 options for what to do when adding the new light and dark theme id's.

1. Upgrade people using the existing theme id's to the new "enhanced" C++ themes.  The "new" themes added will actually have the old theme specifications. (that is what this PR currently does)
2. Require people to switch to the new theme id's to get the new "enhanced" C++ themes.  The existing theme remains unchanged.

If we are doing the first option, our choices for the new id's are "Visual Studio 2017 Light/Dark - C++" or "Visual Studio Classic Light/Dark - C++".  Leaving the VS version out of the "main" theme allows us to update it to whatever we need to in order to stay current with VS.  I think adding "2017" to the name is more future-proof than "Classic" because if another new theme is introduced, we can add a third theme with "2019" in the name to hold the theme introduced in mid-2019.

If we are doing the second option, our choices for the new id's are "Visual Studio 2019 Light/Dark - C++" or "Visual Studio 2022 Light/Dark - C++".  Then the theme with no version number in it is the 2017 theme that we originally released.

I prefer option 1 and naming the new theme with "2017" in the name.

Reply:
I like Bob's option #1 to rename the existing themes as 2017 and not refer to a VS version for the updated themes.

Reply:
I can't wait to test this theme in 1.13.2 :)

Reply:
@H-G-Hristov FYI it ships as the "C/C++ Themes" extension (currently 1.0.0) and not part of the "C/C++" extension (which has 1.13.2 pending).

Reply:
2017 it is! That sounds better than "classic".